ethereum/eth_accounts.yaml

```diff
@@ -19,6 +16,13 @@ components:
         tags: []
         requestBody:
           description: None
+          parameters:
+          - name: apiKey
+            in: path
+            schema:
+              type: string
+              default: demo
+            required: true
```

Comment:
This should be above the requestBody and below tags (not inside the requestBody)